### PR TITLE
Document how to update encrypted Travis credentials.

### PR DESCRIPTION
**Pull request by daithiocrualaoich**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,23 @@ described earlier.
...
 
 
+Travis
+------
+CSearch is continuously integrated on [Travis CI].
+
+To update encrypted credentials in the `.travis.yml` file, use the
+[Travis command line tool].
+
+    gem install travis
```
Comment (passy) [inline README.md:51]: Alternatively, to stick with our spiel here, there's a docker container so you don't have to deal with terrible third-party package managers: https://hub.docker.com/r/caktux/travis-cli/

Comment (passy) [inline README.md:51]: (If we want to do this inside the container though, I'm not sure what the current docker-in-docker story is like.)

**Comment by passy** [comment]
👍

File: Dockerfile [hunk 2]
```diff
@@ -13,6 +13,14 @@ RUN     apt-get install -qq build-essential gcc
 
 
 ################################################################################
+# Travis
+################################################################################
+
+RUN     apt-get install -qq ruby ruby-dev
+RUN     gem install travis
```
Comment (daithiocrualaoich) [inline Dockerfile]: Add `--no-ri --no-rdoc` and also install `travis-lint`

File: README.md (hunk 1 above)
Comment (daithiocrualaoich) [inline README.md:51]: I've merged the travis packages from that container definition into the Dockerfile in this repo now.

**Pull request merged**